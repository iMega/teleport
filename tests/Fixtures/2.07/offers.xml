<?xml version="1.0" encoding="UTF-8"?>
<КоммерческаяИнформация xmlns="urn:1C.ru:commerceml_2" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" ВерсияСхемы="2.07" ДатаФормирования="2015-08-20T19:19:59">
	<Классификатор>
		<Ид>4b86411f-b999-b421-e635-d655ee9fd946</Ид>
		<Наименование>Классификатор (Каталог товаров 4B86411F)</Наименование>
		<Владелец>
			<Ид>53823102-1768-11e5-80cd-6cf049551cd2</Ид>
			<Наименование>Сигма ООО</Наименование>
			<ОфициальноеНаименование>Сигма ООО</ОфициальноеНаименование>
			<ОКПО/>
		</Владелец>
	</Классификатор>
	<ПакетПредложений СодержитТолькоИзменения="false">
		<Ид>4b86411f-b999-b421-e635-d655ee9fd946#</Ид>
		<Наименование>Пакет предложений ( Каталог товаров 4B86411F )</Наименование>
		<ИдКаталога>4b86411f-b999-b421-e635-d655ee9fd946</ИдКаталога>
		<ИдКлассификатора>4b86411f-b999-b421-e635-d655ee9fd946</ИдКлассификатора>
		<Владелец>
			<Ид>53823102-1768-11e5-80cd-6cf049551cd2</Ид>
			<Наименование>Сигма ООО</Наименование>
			<ОфициальноеНаименование>Сигма ООО</ОфициальноеНаименование>
			<ОКПО/>
		</Владелец>
		<ТипыЦен>
			<ТипЦены>
				<Ид>8cd392e1-45b8-11e5-80d0-6cf049551cd2</Ид>
				<Наименование>Бла бла бла</Наименование>
				<Валюта>RUB</Валюта>
				<Налог>
					<Наименование>НДС</Наименование>
					<УчтеноВСумме>true</УчтеноВСумме>
					<Акциз>false</Акциз>
				</Налог>
			</ТипЦены>
		</ТипыЦен>
		<Склады>
			<Склад>
				<Ид>53ce39d7-176b-11e5-80cd-6cf049551cd2</Ид>
				<Наименование>Основной</Наименование>
			</Склад>
		</Склады>
		<Предложения>
			<Предложение>
				<Ид>faf9ca10-176a-11e5-80cd-6cf049551cd2#92a3aba8-176a-11e5-80cd-6cf049551cd2</Ид>
				<Артикул>GU641DWBZ539</Артикул>
				<Наименование>Туалетная вода Gucci Flora Eau Fraiche (30 мл)</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
					<Пересчет>
						<Единица>796</Единица>
						<Коэффициент>1</Коэффициент>
					</Пересчет>
				</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление> 3 500 RUB за PCE</Представление>
						<ИдТипаЦены>8cd392e1-45b8-11e5-80d0-6cf049551cd2</ИдТипаЦены>
						<ЦенаЗаЕдиницу>3500</ЦенаЗаЕдиницу>
						<Валюта>RUB</Валюта>
						<Единица>PCE</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>10</Количество>
				<Склад ИдСклада="53ce39d7-176b-11e5-80cd-6cf049551cd2" КоличествоНаСкладе="10"/>
			</Предложение>
			<Предложение>
				<Ид>faf9ca10-176a-11e5-80cd-6cf049551cd2#72a2167b-434b-11e5-80d0-6cf049551cd2</Ид>
				<Артикул>GU641DWBZ539</Артикул>
				<Наименование>Туалетная вода Gucci Flora Eau Fraiche (50 мл)</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
					<Пересчет>
						<Единица>796</Единица>
						<Коэффициент>1</Коэффициент>
					</Пересчет>
				</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление> 4 200 RUB за PCE</Представление>
						<ИдТипаЦены>8cd392e1-45b8-11e5-80d0-6cf049551cd2</ИдТипаЦены>
						<ЦенаЗаЕдиницу>4200</ЦенаЗаЕдиницу>
						<Валюта>RUB</Валюта>
						<Единица>PCE</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>14</Количество>
				<Склад ИдСклада="53ce39d7-176b-11e5-80cd-6cf049551cd2" КоличествоНаСкладе="14"/>
			</Предложение>
			<Предложение>
				<Ид>faf9ca10-176a-11e5-80cd-6cf049551cd2#ade4adcb-434b-11e5-80d0-6cf049551cd2</Ид>
				<Артикул>GU641DWBZ539</Артикул>
				<Наименование>Туалетная вода Gucci Flora Eau Fraiche (70 мл)</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
					<Пересчет>
						<Единица>796</Единица>
						<Коэффициент>1</Коэффициент>
					</Пересчет>
				</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление> 5 600 RUB за PCE</Представление>
						<ИдТипаЦены>8cd392e1-45b8-11e5-80d0-6cf049551cd2</ИдТипаЦены>
						<ЦенаЗаЕдиницу>5600</ЦенаЗаЕдиницу>
						<Валюта>RUB</Валюта>
						<Единица>PCE</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>0</Количество>
				<Склад ИдСклада="53ce39d7-176b-11e5-80cd-6cf049551cd2" КоличествоНаСкладе="0"/>
			</Предложение>
			<Предложение>
				<Ид>c5211d29-197f-11e5-80cd-6cf049551cd2</Ид>
				<Артикул/>
				<Наименование>Монтаж сети</Наименование>
				<БазоваяЕдиница Код="796" НаименованиеПолное="Штука" МеждународноеСокращение="PCE">
					<Пересчет>
						<Единица>796</Единица>
						<Коэффициент>1</Коэффициент>
					</Пересчет>
				</БазоваяЕдиница>
				<Цены>
					<Цена>
						<Представление> 0 RUB за PCE</Представление>
						<ИдТипаЦены>8cd392e1-45b8-11e5-80d0-6cf049551cd2</ИдТипаЦены>
						<ЦенаЗаЕдиницу>0</ЦенаЗаЕдиницу>
						<Валюта>RUB</Валюта>
						<Единица>PCE</Единица>
						<Коэффициент>1</Коэффициент>
					</Цена>
				</Цены>
				<Количество>0</Количество>
				<Склад ИдСклада="53ce39d7-176b-11e5-80cd-6cf049551cd2" КоличествоНаСкладе="0"/>
			</Предложение>
		</Предложения>
	</ПакетПредложений>
</КоммерческаяИнформация>