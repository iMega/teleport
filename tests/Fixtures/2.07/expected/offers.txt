{"entityType":40,"changes":true}
{"entityType":50,"guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#92a3aba8-176a-11e5-80cd-6cf049551cd2","prod_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","barcode":"","title":"Туалетная вода Gucci Flora Eau Fraiche (30 мл)","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"10","postType":"product_variation"}
{"entityType":70,"offer_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#92a3aba8-176a-11e5-80cd-6cf049551cd2","title":" 3 500 RUB за PCE","price":"3500","currency":"RUB","unit":"PCE","ratio":"1","type_guid":"8cd392e1-45b8-11e5-80d0-6cf049551cd2"}
{"entityType":50,"guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#72a2167b-434b-11e5-80d0-6cf049551cd2","prod_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","barcode":"","title":"Туалетная вода Gucci Flora Eau Fraiche (50 мл)","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"14","postType":"product_variation"}
{"entityType":70,"offer_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#72a2167b-434b-11e5-80d0-6cf049551cd2","title":" 4 200 RUB за PCE","price":"4200","currency":"RUB","unit":"PCE","ratio":"1","type_guid":"8cd392e1-45b8-11e5-80d0-6cf049551cd2"}
{"entityType":50,"guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#ade4adcb-434b-11e5-80d0-6cf049551cd2","prod_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2","barcode":"","title":"Туалетная вода Gucci Flora Eau Fraiche (70 мл)","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"0","postType":"product_variation"}
{"entityType":70,"offer_guid":"faf9ca10-176a-11e5-80cd-6cf049551cd2#ade4adcb-434b-11e5-80d0-6cf049551cd2","title":" 5 600 RUB за PCE","price":"5600","currency":"RUB","unit":"PCE","ratio":"1","type_guid":"8cd392e1-45b8-11e5-80d0-6cf049551cd2"}
{"entityType":50,"guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","prod_guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","barcode":"","title":"Монтаж сети","base_unit":"шт","base_unit_key":"796","base_unit_title":"Штука","base_unit_int":"PCE","amount":"0","postType":"product_variation"}
{"entityType":70,"offer_guid":"c5211d29-197f-11e5-80cd-6cf049551cd2","title":" 0 RUB за PCE","price":"0","currency":"RUB","unit":"PCE","ratio":"1","type_guid":"8cd392e1-45b8-11e5-80d0-6cf049551cd2"}
